source/content/guides/drupal-9-from-community.md

```diff
@@ -0,0 +1,192 @@
+  ```
+
+1. Create an empty `config` folder:
+
+  ```bash{promptUser: user}
+  mkdir -p config
+  ```
+
+## Update Composer Settings
+
+Add/modify the settings in `blob/main/composer.json` as follows:
+
+1. Add the `upstream-configuration` path repository:
+
+   ```
+   "autoload": {
+       "classmap": [
+           "upstream-configuration/scripts/ComposerScripts.php"
+       ]
+   },
+   ```
+
+1. Include the following in the `require` section:
+
+   ```
+   "require": {
+       "pantheon-upstreams/upstream-configuration": "dev-main",
+       "pantheon-systems/drupal-integrations": "^9",
+       "drush/drush": "^11 || ^12"
+   },
+   ```
+
+1. Add `pantheon-systems/drupal-integrations` and quicksilver scripts (optional) to `extra`:
+   
+   ```
+   "extra": {
+       "drupal-scaffold": {
+           "locations": {
+               "web-root": "./web"
+           },
+//highlight-start
+           "allowed-packages": [
+               "pantheon-systems/drupal-integrations"
+           ],
+//highlight-end
+           "file-mapping": {
+               "[project-root]/.editorconfig": false,
+               "[project-root]/pantheon.upstream.yml": false,
+               "[project-root]/.gitattributes": false
+           }
+       },
+       "installer-paths": {
+           "web/core": ["type:drupal-core"],
+           "web/libraries/{$name}": ["type:drupal-library"],
+           "web/modules/contrib/{$name}": ["type:drupal-module"],
+           "web/profiles/contrib/{$name}": ["type:drupal-profile"],
+           "web/themes/contrib/{$name}": ["type:drupal-theme"],
+           "drush/Commands/contrib/{$name}": ["type:drupal-drush"],
+           "web/modules/custom/{$name}": ["type:drupal-custom-module"],
+           "web/profiles/custom/{$name}": ["type:drupal-custom-profile"],
+           "web/themes/custom/{$name}": ["type:drupal-custom-theme"],
+//highlight-start
+           "web/private/scripts/quicksilver/{$name}/": ["type:quicksilver-script"]
+//highlight-end
+       },
+       "composer-exit-on-patch-failure": true,
+       "patchLevel": {
+           "drupal/core": "-p2"
+       }
+   },
+   ```
+1. Add `autoload.classmap`:
+
+   ```
+   "autoload": {
+       "classmap": [
+           "upstream-configuration/scripts/ComposerScripts.php"
+       ]
+   },
+   ``` 
+
+1. Add these `scripts` and `scripts-description` sections: 
+
+   ```
+   "scripts": {
+       "pre-update-cmd": [
+           "DrupalComposerManagedComposerScripts::preUpdate"
+       ],
+       "upstream-require": [
+           "DrupalComposerManagedComposerScripts::upstreamRequire"
+       ]
+   },
+   "scripts-descriptions": {
+       "upstream-require": "Add a dependency to an upstream. See https://pantheon.io/docs/create-custom-upstream or information on creating custom upstreams."
+   },
+   ```
+
+## Update settings.php
+
+Add the following to your `/web/sites/default/settings.php` file.
+
+```
+include __DIR__ . "/settings.pantheon.php";
```

Comment:
Missing steps that I just noticed:

1) Initialize the git repo and commit everything:

`git init -b master`
`git commit -am "Initial commit"`

2) Add the Pantheon repository as a remote:

`git remote add origin <pantheon_remote>`

To get the pantheon_remote you could use terminus:

`terminus connection:info --field=git_url <site>.dev`

3) Force push to Pantheon master branch:

`git push origin master -f`

4) Install your site in the dev environment and test that everything works